checks/redis/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/alexliesenfeld/health/checks/redis
+
+go 1.19
+
+require (
+	github.com/go-redis/redis v6.15.9+incompatible
```

Comment:
The current version is 8 (or 9-beta). Is there a reason we use 6 here?

Reply:
Changed this to v8